low-level/files/opt/seagate/sspl/conf/setup.yaml

```diff
@@ -14,34 +14,38 @@
 # cortx-questions@seagate.com.
 
 sspl:
-    post_install:
-         script: /opt/seagate/cortx/sspl/bin/sspl_setup
-         args:
-            - post_install -p LDR_R2
-    init:
-         script: /opt/seagate/cortx/sspl/bin/sspl_setup
-         args:
-            - init -r cortx
-    config:
-         script: /opt/seagate/cortx/sspl/bin/sspl_setup
-         args:
-            - config -f
-    test:
-         script: /opt/seagate/cortx/sspl/bin/sspl_setup
-         args:
-            - test self
-    reset:
-         script: /opt/seagate/cortx/sspl/bin/sspl_setup
-         args:
-            - reset hard -p LDR_R2
-    ha:
-         script: /opt/seagate/cortx/ha/conf/script/build-ha-sspl
-         args:
-            - /opt/seagate/cortx/iostack-ha/conf/build-ha-args.yaml
-
-    ha-cleanup:
-         script: /opt/seagate/cortx/ha/conf/script/prov-ha-sspl-reset
-         args: null
-
-support_bundle:
-    - /usr/bin/sspl_bundle_generate
+  post_install:
+    cmd: /opt/seagate/cortx/sspl/bin/sspl_setup post_install
+    args: --config <URL>
+  config:
+    cmd: /opt/seagate/cortx/sspl/bin/sspl_setup config
+    args: --config <URL>
+  init:
+    cmd: /opt/seagate/cortx/sspl/bin/sspl_setup init
+    args: --config <URL>
+  test:
+    cmd: /opt/seagate/cortx/sspl/bin/sspl_setup test
+    args: --config <URL> --plan sanity
+  upgrade:
+    cmd: /opt/seagate/cortx/sspl/bin/sspl_setup upgrade
+    args: --config <URL>
+  reset:
+    cmd: /opt/seagate/cortx/sspl/bin/sspl_setup reset
+    args: --config <URL> --type hard
```

Comment:
Can we use `<GLOBAL_CONFIG>` instead of `<URL>`  ?

Reply:
As provisioner already knows what needs to be passed for --config, I'm fine with `<URL>` too.